lib/cursor.js

```diff
@@ -139,7 +139,9 @@ var Cursor = function(bson, ns, cmd, options, topology, topologyOptions) {
     // Promise library
     promiseLibrary: promiseLibrary,
     // Current doc
-    currentDoc: null
+    currentDoc: null,
+    // Optional ClientSession
+    session: options.session || null
```

Comment:
Do we specifically need `null` here, or can we just let it be `undefined`?

Reply:
nope, good catch